shared/js/background/broken-site-report.js

```diff
@@ -147,7 +147,8 @@ export function breakageReportForTab ({
         urlParametersRemoved,
         ctlYouTube,
         ctlFacebookPlaceholderShown,
-        ctlFacebookLogin
+        ctlFacebookLogin,
+        protectionsState: tab.site.isFeatureEnabled('contentBlocking') ? '1' : '0'
```

Comment:
if you follow this method call, you'll see it correctly encodes the logic needed as per the spec in Asana. The extension is unique in that it supports a 'denylist', but that's handled too.